### DISCUSSION TITLE
Some questions about the theory of the cosserat rod.

### DISCUSSION THREAD

**Discussion by Qianjun-Xia** [GazzolaLab/PyElastica#353] [Q&A]
Hi,

I am doing my undergraduate project and it concerns about the cosserat rod.

In the website:https://www.cosseratrods.org/cosserat_rods/theory/, I find there are two equations as $e=\frac{s}{\hat{s}}$ and $\overline{r}_s = e\overline{t}$. In my opinion $\overline{r}_s = \frac{\overline{r}}{\hat{s}}*\frac{\hat{s}}{s}=\frac{1}{e}*t$. I am not sure where is wrong in what I write.

English is not my native language, so if my tone is offensive, I sincerely apologize.

**Comment by armantekinalp** [comment]
Hi @CraYAnshINcHAN,

You have a typo in equation you wrote above e=ds/d\hat{s}, don't forget the differential in front of s. ds term is basically the current length of the rod and d\hat{s} is the rest length of the rod. Since r_s = dr/ds it would be equal to e \cdot t which is dilatation * tangent vector. I also suggest you to go through the paper here is the [link](https://mattia-lab.com/wp-content/uploads/2018/06/Gazzola_RSOS_2018.pdf)

**Comment by Qianjun-Xia** [comment]
It is so kind of you! Sorry I forgot to type all of the \partial, I will go through the paper to find the answer and make myself more clear.
